Samples/Shared/CameraKsPropertyHelper/CameraKsPropertyInquiry.idl

```diff
@@ -0,0 +1,236 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace CameraKsPropertyHelper
+{
+    //
+    // Projection to WinRT of parts of ksmedia.h
+    // 
+    //
+
+    // refer to KSPROPERTY_VIDCAP_CAMERACONTROL
+    enum VidCapCameraControlKind
+    {
+        KSPROPERTY_CAMERACONTROL_PAN = 0,                  // 0

```

Comment:
>KSPROPERTY_CAMERACONTROL_PAN [](http://example.com/codeflow?start=8&length=28)

Is it possible to avoid this by leveraging ksmedia.h in SV2, or is it an unnecessary pain?

Reply:
I'm not aware how we could project these to managed side from the ksmedia.h directly.